### PR TITLE
added forms to sign up and login

### PR DESCRIPTION
**Pull request by franziskacomploi**

### CODE REVIEW

File: frontend/src/components/login.js [hunk 1]
```diff
@@ -1,10 +1,35 @@
 import React from 'react';
 
-const Login = () => (
-    <div>
+const Login = () => {
+    const handleSubmit = (evt) => {
```
Comment (rodgobbi) [inline frontend/src/components/login.js:4]: I think this abbreviation is a bit confusing, usually for events we just use `e` or `event`.
Better usually use the whole word, it's always easier to read.

File: frontend/src/components/login.js [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+                <button type="submit">Log In</button>
+                <input type="reset" value="Reset the filter"></input>
+                </p>
+                
+            </fieldset>
+            </form>
+            </div>
+
+    )
+};
```
Comment (rodgobbi) [inline frontend/src/components/login.js:32]: I have 2 recommendations:
- Have you ever used [prettier](https://prettier.io/)? It's so handy and also the default tool in development nowadays. It formats your code instantly.
- About the formatting options itself, I'd recommend having a indentation of 2 spaces between lines (the tab size), I think it's 4 right now, 4 spaces will easily overflow the editor as the components get more complex. 2 spaces is the default value in `prettier` if you want to use it.

File: frontend/src/components/signup.js [hunk 3]
```diff
@@ -1,10 +1,51 @@
...
-    </div>
-);
+    return (
+            <div>
+            <form method="post" action="/st21-what-the-wine/public/index.html" onSubmit={handleSubmit}>
+            <fieldset>
+            <legend>Sign Up for personalized Wine Recommendations</legend>
+                <p>
+                    <label name="firstname">First Name</label>
+                    <input type="text" name="firstname"></input>
```
Comment (rodgobbi) [inline frontend/src/components/signup.js:16]: One accessibility detail about `label`, you need to use the attribute `for` with the `input`'s `id` value if it's separate fro the target `input`. More info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label
Another approach is wrapping the `input` tag with `label`, this way you don't need to manage ids, which need to be unique in the page.

File: frontend/src/components/login.js (hunk 1 above)
Comment (franziskacomploi) [reply frontend/src/components/login.js:4]: Changed it to e!

File: frontend/src/components/login.js (hunk 2 above)
Comment (franziskacomploi) [reply frontend/src/components/login.js:32]: Tried to look into prettier, need to figure out how that works. Thanks!

File: frontend/src/components/signup.js (hunk 3 above)
Comment (franziskacomploi) [reply frontend/src/components/signup.js:16]: Also if I have a <select> or if I have one label with two inputs?
But I'll try with wrapping the `input` tag with the `label`!

**Pull request merged**